client/src/app/@core/data/maps.service.ts

```diff
@@ -106,7 +114,7 @@ export class MapsService {
   uploadMapFile(uploadLocation: string, mapFile: File): Observable<any> {
     const formData = new FormData();
     formData.append('mapFile', mapFile, mapFile.name);
-    return this.http.post(uploadLocation, formData, {
+    return this.http.put(uploadLocation, formData, {
```

Comment:
In the map-edit.component.ts, we get the current upload location of the map and then upload the new version to the same place, overwriting it.
We could change this so that old versions of the map are kept on the server. Up to the team